src/core/shape/vertex.js

```diff
@@ -609,21 +610,86 @@ p5.prototype.endContour = function() {
  * </code>
  * </div>
  *
+ * @example
+ * <div>
+ * <code>
+ * let fx;
+ * let vs = `#version 300 es
+ *
+ * precision mediump float;
+ *
+ * in vec3 aPosition;
+ *
+ * uniform mat4 uModelViewMatrix;
+ * uniform mat4 uProjectionMatrix;
+ *
+ * void main() {
+ *
+ *   vec4 positionVec4 = vec4(aPosition, 1.0);
+ *   gl_Position = uProjectionMatrix * uModelViewMatrix * positionVec4 -
+ *   vec4(vec2(float(gl_InstanceID) * 100.0, 0.0), 0.0, 0.0);
```

Comment:
Can you add some comments to this shader to explain what's happening here? Maybe there's somewhere we can also add gl_InstanceID to the documentation?